### PR TITLE
New Crowdin translations webapp_consent

### PR DESCRIPTION
**Pull request by LiteFarm-eng**
New Crowdin pull request with translations of webapp_consent

### CODE REVIEW

File: packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ LiteFarm es un proyecto de investigación científica participativa alojado en l
 
 #### ¿Quiénes integran el equipo LiteFarm?
 
-El equipo de LiteFarm es un grupo multidisciplinario de científicos(as), investigadores(as), agricultores(as), diseñadores(as) y profesionales del software. El equipo se formó inicialmente en la Universidad de la Columbia Británica (UBC), Canadá, y ha crecido hasta incluir una red de trabajo colaborativa de individuos y organizaciones. Los investigadores principales son la Dra. Zia Mehrabi (Universidad de Colorado) y la Dra. Hannah Wittman (hannah.wittman@ubc.ca), así como el Dr. Sean Smukler, director del Centro de Sistemas Alimentarios Sostenibles de la UBC Farm.
+El equipo de LiteFarm es un grupo multidisciplinario de científicos(as), investigadores(as), agricultores(as), diseñadores(as) y profesionales del software. El equipo se formó inicialmente en la Universidad de la Columbia Británica (UBC), Canadá, y ha crecido hasta incluir una red de trabajo colaborativa de individuos y organizaciones. Hannah Wittman (hannah.wittman@ubc.ca), así como el Dr. Sean Smukler, director del Centro de Sistemas Alimentarios Sostenibles de la UBC Farm.
```
Comment (Duncan-Brain) [inline packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md:15]: Same problems we saw as before -- need to remove the es, pt and fr.

**Review by Duncan-Brain — COMMENTED** [review]
Cannot merge until we either resole the pt, es, fr translator issues. (missing / not translated sentences). Or keep existing and remove them from this pr.

File: packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md (hunk 1 above)
Comment (kathyavini) [reply packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md:15]: I think I can push that as a change to this branch directly, let's see!

Comment (kathyavini) [reply packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md]: @Duncan-Brain okay cleanly done! I'd like to merge the others please so I can include consent in the language-switching framework Pr 🙏

**Review by kathyavini — APPROVED** [review]

**Review by Duncan-Brain — APPROVED** [review]

**Pull request merged**